src/db.rs

```diff
@@ -56,6 +59,59 @@ pub fn get_dbconn() -> mysql::Result<PooledConn> {
     Ok(dbconn)
 }
 
+pub fn db_exists() -> anyhow::Result<()> {
+    let mut dbconn = get_dbconn()?;
+
+    let dbname = env::var("KBS_DB").expect("KBS_DB not set");
+
+    let dbres: Vec<String> = dbconn.query("SHOW DATABASES")?;
+
+    if dbres.contains(&dbname) {
+        return Ok(());
+    }
+    Ok(())
+}
+
+pub fn drop_kbs_db() -> anyhow::Result<()> {
+    let host_name = env::var("KBS_DB_HOST").expect("KBS_DB_HOST not set");
+    let user_name = env::var("KBS_DB_USER").expect("KBS_DB_USER not set.");
+    let db_name = env::var("KBS_DB").expect("KBS_DB not set.");
+    let db_pw = env::var("KBS_DB_PW").expect("KBS_DB_PW not set.");
+
+    const DBEX: &str = "/usr/bin/mysql";
```

Comment:
might not be wise to assume the path of `usr/bin/mysql`

Reply:
I deleted this funciton.